### ISSUE TITLE
Proposal: make async CF polling more available

### ISSUE THREAD

**Issue by arschles** [deis/steward#226]
After #222, only a single steward will be responsible for doing the polling after an asynchronous provision. This proposal is for discussing whether it's worth distributing the polling work. Doing so would increase the reliability (AKA availability) of the polling work, but will add more complexity.

**Comment by arschles** [comment]
Closing in favor of https://github.com/deis/steward-framework/issues/39#issuecomment-260794234
